### PR TITLE
[CL-511] - add variant to links within banners

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/CL-511

## 📔 Objective

This PR updates link `linkType` prop from `contrast` to `secondary` for all components whose banners include a link, fixing a poor color contrast.

## 📸 Screenshots

Screenshot 2024-11-21 at 1 07 11 PM

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by Jingo88 — APPROVED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Review by mzieniukbw — APPROVED** [review]
LGTM from SM side

File: apps/web/src/app/vault/individual-vault/vault-banners/vault-banners.component.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <a
     bitLink
     linkType="contrast"
+    variant="secondary"
```
Comment (shane-melton) [inline apps/web/src/app/vault/individual-vault/vault-banners/vault-banners.component.html:13]: ⚠️ `variant` is not a supported input for `bitLink`. I think you want to update the `linkType` attribute to be `secondary` instead.

e.g. `linkType="secondary"`

File: apps/web/src/app/billing/organizations/payment-method/organization-payment-method.component.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
   {{ freeTrialData.message }}
   <a
     bitLink
-    linkType="contrast"
+    link="secondary"
```
Comment (shane-melton) [inline apps/web/src/app/billing/organizations/payment-method/organization-payment-method.component.html:12]: ⚠️ `link` is not a supported attribute of `bitLink`. I think you want to still use the `linkType="secondary"`

**Review by shane-melton — CHANGES_REQUESTED** [review]
I think we just have a slight mixup of attribute names here that we need to fix.

**Review by shane-melton — APPROVED** [review]

**Review by mzieniukbw — APPROVED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Review by Jingo88 — APPROVED** [review]

**Pull request merged**